swisscom_ai/research_keyphrase/model/input_representation.py

```diff
@@ -9,7 +9,7 @@
 class InputTextObj:
     """Represent the input text in which we want to extract keyphrases"""
 
-    def __init__(self, pos_tagged, lang, stem=False, min_word_len=3):
+    def __init__(self, pos_tagged, lang, stem=False, min_word_len=1):
```

Comment:
Nitpicking: in general I would not change the values of default parameter in a method's definition when refactoring, this tends to silently break behavior for people using your library/fork. Instead, I would pass another value everywhere you call the function. In our case it's not too important since we only use it in one project :)

Reply:
Good point :)